onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -502,6 +502,12 @@ void BackendManager::Compute(OrtKernelContext* context) {
 
     dynamic_backend->Infer(context);
   } else {
+    if (dynamic_workload_type == "EFFICIENT") {
```

Comment:
Also needs implementation for Setting Dynamic Option -> ep.dynamic.workload_type to "DEFAULT"

Reply:
Done